h/views/api/decorators/response.py

```diff
@@ -4,17 +4,25 @@
 from __future__ import unicode_literals
 
 from h.views.api import API_VERSION_DEFAULT
-from h.views.api.helpers.media_types import media_type_for_version
+from h.views.api.helpers.media_types import media_type_for_version, version_media_types
 
 
 def version_media_type_header(wrapped):
     """View decorator to add response header indicating API version"""
 
     def wrapper(context, request):
         response = wrapped(context, request)
-        # TODO: expand with logic for multiple versions once the app knows
-        # about multiple versions
+        # Assume default version...
```

Comment:
The previous variant of this wrapper was dumb (on purpose at the time) and responded with the media type for the default API version, because there was only one version, and it was the default. Now it will respond more smarter.